DataFormats/L1THGCal/src/classes_def.xml

```diff
@@ -21,16 +21,16 @@
   <class name="edm::Wrapper<l1t::HGCalTowerBxCollection>"/>
 
   <class name="l1t::HGCalClusterT<l1t::HGCalTriggerCell>" />
-  <class name="l1t::HGCalCluster" ClassVersion="10">
-    <version ClassVersion="10" checksum="607544984"/>
+  <class name="l1t::HGCalCluster" ClassVersion="11">
+    <version ClassVersion="11" checksum="4174333084"/>
```

Comment:
"updateclassversion" does now work for me. Do we really need to keep the checksums? Any hostile code insertion possibilities?